crates/polars-core/src/config.rs

```diff
@@ -33,6 +33,10 @@ pub fn verbose() -> bool {
     std::env::var("POLARS_VERBOSE").as_deref().unwrap_or("") == "1"
 }
 
+pub fn get_default_engine() -> String {
+    std::env::var("POLARS_DEFAULT_ENGINE").unwrap_or_else(|_| "cpu".to_string())
+}
+
```

Comment:
Just a question: why even change polars-core when changing the engine is a Python-specific feature?

Reply:
I could see this feature being used in other bindings.